optimum/exporters/onnx/model_configs.py

```diff
@@ -17,7 +17,7 @@
 from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple, Union
 
 from packaging import version
-from transformers.utils import is_tf_available
+from transformers.utils import is_tf_available, is_torch_available
```

Comment:
Is `is_torch_available` used anywhere here?

Reply:
ah sorry, forgot about that
I was trying to check torch available then torch version, but I saw that there is MIN_TORCH_VERSION variable